### PR TITLE
6 submit an application

### LINKED ISSUE #7: 6. Submit an Application
As a visitor
When I visit an application's show page
And I have added one or more pets to the application
Then I see a section to submit my application
And in that section I see an input to enter why I would make a good owner for these pet(s)
When I fill in that input
And I click a button to submit this application
Then I am taken back to the application's show page
And I see an indicator that the application is "Pending"
And I see all the pets that I want to adopt
And I do not see a section to add more pets to this application

### PR DESCRIPTION
**Pull request by DavisWeimer**
- User Story 6 functionality built tested and passing
 - Many-to-many migration re-generated
 Close #7

### CODE REVIEW

**Review by ConnorRichmond — APPROVED** [review]
Killer

File: app/controllers/adoption_apps_controller.rb [hunk 1]
```diff
@@ -37,26 +37,27 @@ def search_pets
   end
 
   def update
-    created = "In Progress"
```
Comment (dani-wilson) [inline app/controllers/adoption_apps_controller.rb:40]: Note to myself to take this out later

File: app/views/application/show.html.erb [hunk 2]
```diff
@@ -11,10 +11,12 @@
 <h3>Pet Names: <%= @adoption_app.pet_names %></h3>
 <h3>Status: <%= @adoption_app.status %></h3>
 
-<%= form_with url: "/adoption_apps/#{@adoption_app.id}/search_pets", method: :get do |form| %>
-  <%= form.label :search, "Search Pets by Name" %>
-  <%= form.text_field :search, placeholder: "Search Pets by Name" %>
-  <%= form.submit "Submit" %>
+<% if @adoption_app.status != "Pending" %>
```
Comment (dani-wilson) [inline app/views/application/show.html.erb:14]: We ❤️ conditionals!

**Review by dani-wilson — APPROVED** [review]
Looks great! Yay!

**Pull request merged**